ansible/playbook/tasks/update.yml

```diff
@@ -169,6 +169,18 @@
         option: external_snapshot_name
         value: Share with Percona
 
+    - name: Set Grafana folder for plugins on /srv partition
+      ini_file:
+        dest: /etc/grafana/grafana.ini
+        section: paths
+        option: plugins
+        value: /srv/grafana/plugins
+
+    - name: Set Grafana folder for plugins on /srv partition for all users
+      lineinfile:
+        path: /etc/bashrc
+        line: 'export GF_PLUGIN_DIR=/srv/grafana/plugins'
```

Comment:
This is unnecessary since it duplicates a much stronger setting in https://github.com/percona/pmm-server/pull/307/files#diff-500a0e8dcb2879d903640ba46031362cdc923d030e278cf2a681c24765cc87b0R67

Reply:
Plugins are installed by cli utility. This utility is using GF_PLUGIN_DIR variable for getting  a path to the Grafana plugin directory

![Screenshot 2021-09-17 at 15 59 00](https://user-images.githubusercontent.com/25184101/133787480-6afbd76b-25ae-4cc1-b65a-b4b2ac534e0f.png)

It has been tested.
![Screenshot 2021-09-17 at 16 04 44](https://user-images.githubusercontent.com/25184101/133787563-de1cfcdf-0966-494c-ab29-0195c57fde66.png)

Reply:
was it tested not in bash? as part of update process

Reply:
It was tested as update from current to current version.

Reply:
So you checked that grafana uses new paths, I'm asking because we don't delete old path in this PR and just copy files from old path, so grafana can keep using old path and work correctly.

Reply:
I have added an additional task for deleting the old folder. 
Here is a result of my test with using the new playbook.

- original state of PMM 2.21.0
![Screenshot 2021-09-22 at 18 09 05](https://user-images.githubusercontent.com/25184101/134372146-ef7ad008-d1f1-40ae-9a7b-1304750a1968.png)

- state after the playbook run
![Screenshot 2021-09-22 at 18 15 50](https://user-images.githubusercontent.com/25184101/134372548-4e64d1d4-2cbb-4a51-a81c-a2da06cb40b9.png)